packages/@react-spectrum/avatar/src/Avatar.tsx

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2021 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {classNames, dimensionValue, useStyleProps} from '@react-spectrum/utils';
+import {FocusRing} from '@react-aria/focus';
+import React, {useRef} from 'react';
+import {SpectrumAvatarProps} from '@react-types/avatar';
+import styles from '@adobe/spectrum-css-temp/components/avatar/vars.css';
+import {useAvatar} from '@react-aria/avatar';
+import {useProviderProps} from '@react-spectrum/provider';
+
+/**
+ * An avatar is a thumbnail representation of an entity, such as a user or an organization.
+ */
+export function Avatar(props: SpectrumAvatarProps) {
+  const {
+    isDisabled,
+    size = 'avatar-size-100',
```

Comment:
I think we can shorten these to their size alone and omit the `avatar-size-` prefix
https://spectrum.adobe.com/page/avatar/#Table-of-options
I could be persuaded to keep the `size-` part since that's how other dimension values are entered in props

Reply:
actually, do avatar sizes match size?

Reply:
+1 to this. Judging from the changes made the global/med/large css files, avatar sizes != spectrum sizes (e.g. `avatar-size-700` = `size-500`). IMO it would easier if we simply redefined the size range provided by Spectrum to be consistent with the standard global sizes (e.g. the valid size value we accept are `size-200` to `size-500`).

Reply:
hah, unlikely that we're ok with a 1px difference, i think we'll need to bring this up with design and find out how to communicate this best

Reply:
i did just notice that this is a `size` prop, not `width`, so this may be less of an issue
though maybe we need to remove width/height as props that Avatar accepts?